io/io/src/RRawFileUnix.cxx

```diff
@@ -108,6 +112,23 @@ void ROOT::Internal::RRawFileUnix::OpenImpl()
    }
 }
 
+void ROOT::Internal::RRawFileUnix::ReadVImpl(RIOVec *ioVec, unsigned int nReq)
+{
+#ifdef R__HAS_URING
+   if (!RIoUring::IsAvailable()) {
+      R__DEBUG_HERE("RRawFileUnix") <<
```

Comment:
Also here we need to use TError for the time being

Reply:
Resolved by 41858029cba8cc7c55d4bf9ae1440f8269fe091a